xbmc/windowing/win10/WinSystemWin10.cpp

```diff
@@ -316,11 +316,14 @@ bool CWinSystemWin10::ChangeResolution(const RESOLUTION_INFO& res, bool forceCha
   {
     bool changed = false;
     auto hdmiInfo = HdmiDisplayInformation::GetForCurrentView();
+    const bool needHDR = DX::DeviceResources::Get()->IsHDROutput();
+
     if (hdmiInfo != nullptr)
     {
       // default mode not in list of supported display modes
-      if (res.iScreenWidth == details->ScreenWidth && res.iScreenHeight == details->ScreenHeight
-        && fabs(res.fRefreshRate - details->RefreshRate) <= 0.00001)
+      if (!needHDR && res.iScreenWidth == details->ScreenWidth &&
+          res.iScreenHeight == details->ScreenHeight &&
+          fabs(res.fRefreshRate - details->RefreshRate) <= 0.00001)
       {
         Wait(hdmiInfo.SetDefaultDisplayModeAsync());
```

Comment:
Is the default mode always SDR?

Reply:
Yes, default mode is 4K / 59,94 Hz / SDR anyway SetDefaultDisplayModeAsync not supports HDR extensions then can not be used to set any HDR mode. 

Then if `needHDR` is true this is not used and is used the other call (RequestSetCurrentDisplayModeAsync) even if mode was the default (which it is not).

Reply:
ok